source/js/Directions.js

```diff
@@ -20,31 +124,114 @@ class Directions extends HTMLElement {
+      
+      text_button.addEventListener("click", event => {
+        if(text_button.classList.contains("listItemStyle")){
+          text_button.classList.remove("listItemStyle");
+          text_button.classList.add("listItemStyleShown");
+          wrapper.removeChild(down_arrow);
+          wrapper.appendChild(up_arrow);
+        }
+        else{
+          text_button.classList.remove("listItemStyleShown");
+          text_button.classList.add("listItemStyle");
+          wrapper.removeChild(up_arrow);
+          wrapper.appendChild(down_arrow);
+        }
+      });
+      
+      list.appendChild(direction);
+     
       // If there are inner steps, display them as well
       if (directionList[i].itemListElement != undefined) {
         for (let j = 0; j < directionList[i].itemListElement.length; j++) {
-          const innerListItem = document.createElement("li");
-          innerListItem.textContent = `${directionList[i].itemListElement[j].text}`;
-          list.appendChild(innerListItem);
-        }
+          let direction = document.createElement("div");
+          let y = j+1;
+          let text_button = document.createElement('button');
+          let wrapper = document.createElement("div");
+          text_button.appendChild(wrapper);
+          let cb = document.createElement("input");
+          cb.type = "checkbox";
+          cb.classList.add("checkbox");
+          direction.appendChild(cb);
+          direction.appendChild(text_button);
+          direction.classList.add("direction");
+          wrapper.classList.add("buttonstyle");
+          let text = document.createElement("p");
+          let down_arrow = document.createElement("img");
+          down_arrow.setAttribute("src", "./images/arrow-down.png");
+          let up_arrow = document.createElement("img");
+          up_arrow.setAttribute("src", "./images/arrowUp.png");
+          down_arrow.classList.add("downArrow");
+          up_arrow.classList.add("upArrow");
+      
+          text.textContent = `${x + "." + y +")"+ " " + directionList[i].itemListElement[j].text}`;
+          text.classList.add("text");
+          wrapper.appendChild(text);
+          wrapper.appendChild(down_arrow);
+          text_button.appendChild(wrapper);
+          text_button.classList.add("listItemStyle");
+          text_button.addEventListener("click", event => {
+            if(text_button.classList.contains("listItemStyle")){
+              text_button.classList.remove("listItemStyle");
+              text_button.classList.add("listItemStyleShown");
+              wrapper.removeChild(down_arrow);
+              wrapper.appendChild(up_arrow);
+            }
+            else{
+              text_button.classList.remove("listItemStyleShown");
+              text_button.classList.add("listItemStyle");
+              wrapper.removeChild(up_arrow);
+              wrapper.appendChild(down_arrow);
+            }
+          });
+      
+        list.appendChild(direction);
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 10 spaces but found 8.](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8460933)
```suggestion
          list.appendChild(direction);
```